pages/_app.js

```diff
@@ -20,16 +42,21 @@ export default function App({ Component, pageProps }) {
         <link rel="icon" href="/favicon.ico" />
       </Head>
       <GlobalStyle />
-      <StyledHeadline>Die Essensretter</StyledHeadline>
+      <StyledHeadline>
+        <Image src={logo} alt="logo" width={300} height={70} />
```

Comment:
In the alt tag you could be a little more specific, what kind of logo?

Reply:
ok I precised it :)